### PR TITLE
pytext recipe

### PR DESCRIPTION
**Pull request by deepali-c**
Add pytext recipe

### CODE REVIEW

**Comment by hartb** [comment]
Thank you, @deepali-c! This looks pretty good to me. Just one question...  Do we still need the patch to revert BERT now that we have `fairseq` in the channel? Does reverting BERT still save us needing `pytorch-transformers` as well?

Other than that I think the pytext content is OK (presumably we'll look to remove some of the patches as some of the PRs are merged upstream).

Maybe @jayfurmanek look more closely at the build machinery. Looked OK to me, but I could easily miss something.  I guess one question as far as that stuff goes... Do we need to separately track licensing for `pytext` itself vs the recipe?

File: conda-recipes/pytext-feedstock/recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - torchtext=0.4.0
+    - typing-extensions=3.7.2
+    - pytorch=1.2.0
+    - tensorboard=1.15.* 
+    - absl-py=0.7.1 
+
+about:
+  home: https://github.com/facebookresearch/pytext 
+  license: BSD
+  license_family: BSD
```
Comment (jayfurmanek) [inline conda-recipes/pytext-feedstock/recipe/meta.yaml]: We need the license file included here.
It needs to point to a file inside the source package or to a license file in the recipe directory.

Comment (deepali-c) [reply conda-recipes/pytext-feedstock/recipe/meta.yaml]: oh yes, missed that.

Comment (deepali-c) [reply conda-recipes/pytext-feedstock/recipe/meta.yaml]: updated it now.

**Comment by deepali-c** [comment]
@hartb, yes as of now I have just taken the recipe that we had at our repo https://github.ibm.com/mldlppc/pytext-feedstock/ and built here. 

Changes for BERT are needed (should have mentioned that earlier) and I will add it may be in seperate PR and it would need testing too.

**Comment by deepali-c** [comment]
I was trying to add test stage to `import pytext` but as expected it fails with the following error:

```
import: 'pytext'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/test_tmp/run_test.py", line 2, in <module>
    import pytext
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/pytext/__init__.py", line 10, in <module>
    from pytext.task.new_task import NewTask
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/pytext/task/__init__.py", line 4, in <module>
    from .new_task import NewTask, _NewTask
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/pytext/task/new_task.py", line 7, in <module>
    from pytext.config import ConfigBase, PyTextConfig
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/pytext/config/__init__.py", line 6, in <module>
    from .serialize import config_from_json, config_to_json, pytext_config_from_json  # noqa
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/pytext/config/serialize.py", line 9, in <module>
    from .component import Registry
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/pytext/config/component.py", line 7, in <module>
    import torch
  File "/home/conda/feedstock_root/build_artifacts/pytext-nlp_1576486446768/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/torch/__init__.py", line 81, in <module>
    from torch._C import *
ImportError: libcuda.so.1: cannot open shared object file: No such file or directory
Tests failed for pytext-nlp-0.2.2-py36_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING:conda_build.build:Tests failed for pytext-nlp-0.2.2-py36_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING conda_build.build:tests_failed(2231): Tests failed for pytext-nlp-0.2.2-py36_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
TESTS FAILED: pytext-nlp-0.2.2-py36_0.tar.bz2```

**Pull request merged**